drivers/include/lis2dh12.h

```diff
@@ -116,11 +116,60 @@ enum {
     LIS2DH12_NODEV = -2,            /**< unable to talk to device */
 };
 
+/**
+ * @brief    interrupt section
+ */
+typedef struct {
+    uint8_t cfg;
+    uint8_t ths;
+    uint8_t duration;
+    uint8_t type;
+} int_params_t;
+
+#define INT_1 0x1
+#define INT_2 0x2
+
 /**
  * @brief   Export the SAUL interface for this driver
  */
 extern const saul_driver_t lis2dh12_saul_driver;
 
+/**
+ * @brief   Set the Interruptvalues in LIS2DH12 sensor device
+ *
+ * @param[in] dev      device descriptor
+ * @param[in] params   device interrupt configuration, with .type, .cfg, .ths and .duration
+ * @param[in] INT_X    number of interrupt line 
+ *
+ * @return  LIS2DH12_OK on success
+ * @return  LIS2DH12_NOBUS on bus errors
+ */
+int lis2dh12_set_interrupt(const lis2dh12_t *dev, int_params_t params, uint8_t INT_X);
```

Comment:
I would suggest abbreviating to `lis2dh12_set_int` because this abbreviation is commonly used in function names in RIOT.